tests/modules/bwa/mem/nextflow.config

```diff
@@ -2,4 +2,8 @@ process {
 
     publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
 
+    withName: BWA_MEM {
+        ext.args2 = { sort_bam ? "" : "-bh" }
```

Comment:
Why is this not the default for both?

Reply:
samtools view and samtools sort don't have the same options

Reply:
output is bam by default for example

Reply:
(and decided by the output name)

Reply:
Ah, right, and `sort` automatically autodetects the format from the file ending right?